database/migrations/000032_provider_class.up.sql

```diff
@@ -0,0 +1,40 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+CREATE TYPE provider_class AS enum ('github', 'gh-app');
```

Comment:
I should have mentioned this earlier, but I'm thinking `github-app` is a better name than `gh-app`. If a user has to decide what class of provider they want to enroll, it's probably easier for them to understand what `github-app` is.

Reply:
I think so too, let's change it.